assets/css/admin/addon-list.css

```diff
@@ -0,0 +1,58 @@
+.el-container {
+	width: 100%;
+}
+
+.clear {
```

Comment:
I did a whole project search and was not able to find the admin css file containing the same css class. Could you please let me know the file name?